app/frontend/src/components/Button.tsx

```diff
@@ -9,7 +9,9 @@ import React, { ElementType } from "react";
 import classNames from "classnames";
 
 const useStyles = makeStyles({
-  root: {},
+  root: {
+    borderRadius: 20,
```

Comment:
Will it always be 20? If so you could do theme.shape.borderRadius and change it to 20 in the theme

Reply:
Oh, I just saw the other line which is 10. What about `theme.shape.borderRadius*2`? Or maybe that's silly. In that case maybe we should remove from the theme file.

Reply:
Yeah I'm not sure. When trying to replicate some default material-ui components, if we use `theme.spacing()` and 8px grid, we have to use some fraction values. I think this is a discussion with the design team later on. Maybe it was too early to try to enforce defaults. 

Looking at the mid-fi again, it seems like cards radius is also 20. And just read about [Material Design guidelines](https://material.io/design/shape/applying-shape-to-ui.html#baseline-shape-values) for different border radius values for small/med/large components. So maybe `theme.shape.borderRadius*2` is a good idea.